pol-core/bscript/compiler/ast/ClassBody.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "bscript/compiler/ast/Statement.h"
+
+namespace Pol::Bscript::Compiler
+{
+class NodeVisitor;
+// Only contains user functions as children, as variable statements are added in
+// `top_level_statements` by SourceFileProcessor.
+class ClassBody : public Statement
+{
+public:
+  ClassBody( const SourceLocation&, NodeVector statements );
```

Comment:
This whole class will change in scoping PR. Currently, it does not hold var statements, and only holds UserFunction AST nodes. This is incorrect, as this approach will create UserFunction nodes for all child functions, even those that are not visited.